lib/core/engine.js

```diff
@@ -103,21 +103,13 @@ class Engine {
 
   pipelineService(_options) {
     const self = this;
-    this.events.emit("status", "Building Assets");
-    const Pipeline = require('../pipeline/pipeline.js');
-    const pipeline = new Pipeline({
-      env: this.env,
-      buildDir: this.config.buildDir,
-      contractsFiles: this.config.contractsFiles,
-      assetFiles: this.config.assetFiles,
-      events: this.events,
-      logger: this.logger,
-      plugins: this.plugins,
+    this.registerModule('pipeline', {
       webpackConfigName: this.webpackConfigName
     });
+    this.events.emit("status", "Building Assets");
```

Comment:
should this event being emitted when we actually send:
`self.events.request('pipeline:build'` ?

Reply:
This is an emission I actually haven't introduced in this PR, it only looks like it because I'm moving stuff around. To be honest I don't know what the `status` event is for...

Reply:
`Building Assets` doesn't show up during `embark run` even with `--loglevel trace` (checked on develop branch). Since we already log messages such as `compiling solidity contracts...` and `running webpack with 'development' config...` perhaps we can just remove it? @iurimatias